examples/vue-example-app/src/components/MapLayout.vue

```diff
@@ -7,7 +7,7 @@
       :minQueryLength="8"
       :showSuburb="false"
       :showMetroBoundary="true"
```

Comment:
If we do need to turn off the boundary altogether then this will need to change.